upstairs/src/downstairs.rs

```diff
@@ -2911,13 +2930,13 @@ impl Downstairs {
                     None,
                 )
             }
-            Message::ReadResponse {
-                upstairs_id,
-                session_id,
-                job_id,
-                responses,
-            } => {
-                cdt::ds__read__io__done!(|| (job_id.0, client_id.get()));
```

Comment:
where did the `read_io_done` probe go?

Reply:
...it's right here?  Currently at L2934:

```rust
  Message::ReadResponse { header, data } => {
      cdt::ds__read__io__done!(|| (header.job_id.0, client_id.get())); // HERE IT IS
      let upstairs_id = header.upstairs_id;
      let session_id = header.session_id;
      let job_id = header.job_id;
      let responses = header.into_read_responses(data);
```

Reply:
ah, there you are!